Create witness.md (#1758)
MicrosoftDocs/azure-stack-docs: azure-stack/hci/manage/witness.md

You can either use an SMB file share as a witness or an Azure cloud witness. An Azure cloud witness is recommended, provided all server nodes in the cluster have a reliable internet connection. This article covers creating a cloud witness.

> Cloud witness uses the HTTPS default port 443 to establish outbound communication with the Azure blob service. Ensure that the HTTPS port is accessible. Azure uses **.core.windows.net** as the Endpoint. Hence you need to have it included in any firewall allow lists you're using between the cluster and Azure Storage.

Reviewer: ```suggestion
Cloud witness uses the HTTPS default port 443 to establish outbound communication with the Azure Blob Storage service. Make sure that port 443 is open in your firewalls and that `*.core.windows.net` is included in any firewall allow lists you're using between the cluster and Azure Storage.
```

Author: Thanks @JasonGerend. Yes the changes looks good
